compiler/rustc_mir_transform/src/coverage/mod.rs

```diff
@@ -189,27 +198,89 @@ fn create_mappings<'tcx>(
             Some(Mapping { kind, code_region })
         },
     ));
+    let mut term_for_bcbs = |bcbs: &[BasicCoverageBlock]| {
```

Comment:
Nitpick: I think this should be renamed to something like `term_for_sum_of_bcbs`.